BrowserServicesKit/Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -155,8 +155,12 @@ public final class DefaultSubscriptionManagerV2: SubscriptionManagerV2 {
     public var tokenRecoveryHandler: TokenRecoveryHandler?
     public let currentEnvironment: SubscriptionEnvironment
     private let isInternalUserEnabled: () -> Bool
-    private var v1MigrationNeeded: Bool = true
+
+    @MainActor
+    private var v1MigrationNeeded = true
```

Comment:
You can ignore the `@MainActor` annotation for now.

What's important here is that `v1MigrationNeeded` can be run by main app, menu app, and isn't thread safe.